test/RestApi.test.ts

```diff
@@ -0,0 +1,25 @@
+import { Template } from '@aws-cdk/assertions';
+import * as cdk from 'aws-cdk-lib';
+import * as MainStack from '../lib/MainStack';
+import * as RestApi from '../lib/RestApi';
+import { Stack } from '@aws-cdk/core';
+
+describe('REST API Test', () => {
+
+    const app = new cdk.App();
+    const stack = new MainStack.DemosCdkApiPipelineStack(app, 'MyTestStack');
+    const assert = Template.fromJSON(app.synth().getStackArtifact(stack.artifactId).template);
+
+    test('REST API Endpoint == 1', () => {
+        expect(assert.findResources('AWS::ApiGateway::RestApi').length).toBe(1); // Find the number of REST API endpoints
```

Comment:
There's a native counter in the assert library for this

`assert.resourceCountIs('Foo::Bar', 2);`

Reply:
Updated. Please have a look.